src/web/routes/application/check/create-request-body.js

```diff
@@ -37,7 +38,8 @@ const createDeviceFingerprint = (headers) => ({
 
 const createRequestBody = (req) => ({
   claimant: createClaim(req.session.claim),
-  deviceFingerprint: createDeviceFingerprint(req.headers)
+  deviceFingerprint: createDeviceFingerprint(req.headers),
+  webUIVersion: APP_VERSION
```

Comment:
Ideally `APP_VERSION` would be passed into the function rather than importing the config:
```
const createRequestBody = (config, req) => ({
  claimant: createClaim(req.session.claim),
  deviceFingerprint: createDeviceFingerprint(req.headers),
  webUIVersion: config.environment.APP_VERSION
})
```

Reply:
done